### DISCUSSION TITLE
Qucs-S v24.4.0

### DISCUSSION THREAD

**Discussion by ra3xdh** [ra3xdh/qucs_s#1044] [Announcements]
Qucs-S v24.4.0 has been released on October, 31. The release contains new features and bugfixes. Here is the summary of the changes:

## New features

* Implemnted touchstone (S2P) files viewer tool #936
* VC resistor made available for SPICE and ADMS-independent #959

## Library update

* Added SPICE_Tline library containing transmission lines for Ngspice #896

## Bugfixes

* Don't resolve paths to relaive if document is not saved and not belong to project #951
* Allow variables for potentiometer and diode device properties #1021 #1013
* Fixed simulation properties editing #968
* Fixed disbale simulation for XYCE #965
* Fixed running simulation in CLI mode #962
* Fixed memory leaks #960
* Fixed BJTsub and diode devices #983
* Fixed Schottky diode symbol appearence #1027
* Fixed crash if no simulator found on the first start #979
* Fixed artifacts when selecting elliptic arc #987
* Fixed text rendiering for Qt6.8.0 #1002
* Fixed ASCO simulation #1035 
* Fixed context menu on the tab diagram #1033 

## QucsatorRF

* QucsatorRF updated to v1.0.3. See the release notes at QucsatorRF repository

**Comment by tomhajjar** [comment]
Web page has wrong ver#. 

I assume the nightly includes everything in the latest release?

![2024-11-05_154258](https://github.com/user-attachments/assets/39796092-e4a7-4824-8e4b-d4a62930ce5f)
